internal/docs/pubsub/design.md

```diff
@@ -417,10 +375,10 @@ type Message struct {
 type AckID interface{}
 
 // Ack acknowledges the message, telling the server that it does not need to
-// be sent again to the associated Subscription. This method blocks until
-// the message has been confirmed as acknowledged on the server, or failure
-// occurs.
-func (m *Message) Ack(ctx context.Context) error {
+// be sent again to the associated Subscription. This method returns
+// immediately and the ack is sent to the server in a separate goroutine
```

Comment:
The phrasing here seems fine for a design doc, but not for a doc comment. To denote the difference, let's move the detail comments inside the function body.

Reply:
Done.